app/views/NewServerView.js

```diff
@@ -35,27 +35,25 @@ const styles = StyleSheet.create({
 		alignSelf: 'center'
 	},
 	inputContainer: {
+		flex: 1,
 		marginTop: 25,
-		marginBottom: 15
-	},
-	input: {
```

Comment:
not sure why you changed the styles and removed sharedStyles?

Reply:
input style was not being used previously, I just have created a new style here and named it input